app/views/middleware_server/_add_jdbc_driver.html.haml

```diff
@@ -0,0 +1,136 @@
+- @angular_form = true
+
+%ng-form.form-horizontal#form_jdbc_add{"name"          => "jdbcAddForm",
+                                       "ng-controller" => "mwAddJdbcDriverController"}
+  .modal.fade#modal_jdbc_div{"tabindex"         => "-1",
+                             "role"             => "dialog",
+                             "aria-labelledby"  => "mw_jdbc_driver_add_label",
+                             "aria-describedby" => "modal",
+                             "aria-hidden"      => "true",
+                             "data-keyboard"    => "false",
+                             "data-backdrop"    => "static",
+                             :style             => "display: none"}
```

Comment:
If this `display:none` is needed, it's probably badly used `.modal`. Patternfly modals are hidden by default.